css/all.css

```diff
@@ -5,6 +5,8 @@
     --hover-color: hsl(var(--color-hue), 55%, 40%);
     --link-color: var(--hover-color);
     --border-color: hsl(var(--color-hue), 60%, 25%);
+    --color-on-background: white;
+    --text-color: #333;

```

Comment:
Can you change the name of this variable to `--dark-text-color`? Text color conflicts with one of our variables in the modern engine, and this would cause strange behavior.